e2e-tests/tests/gr-one-pod/06-read-data.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 30
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      wait_cluster_consistency_gr "${test_name}" "1" "1"
+
+      data=$(run_mysql "SELECT * FROM myDB.myTable" "-h $(get_router_service $(get_cluster_name)) -uroot -proot_password")
+      kubectl create configmap -n "${NAMESPACE}" 06-read-data-minio --from-literal=data="${data}"
+    timeout: 120
```

Comment:
```suggestion
      timeout: 120
```

Reply:
I think it needs to be indented this way to be applied to `script` block.